UPDATING.md

```diff
@@ -1164,6 +1164,54 @@ The goal of this change is to achieve a more consistent and configurale cascadin
 
 When initializing a Snowflake hook or operator, the value used for `snowflake_conn_id` was always `snowflake_conn_id`, regardless of whether or not you specified a value for it. The default `snowflake_conn_id` value is now switched to `snowflake_default` for consistency and will be properly overriden when specified.
 
+### Simplify the response payload of endpoints /dag_stats and /task_stats
```

Comment:
We don't actually need to put anything in here - this is an internal/undocumented API.

Doesn't hurt to have it I guess, just an FYI I guess

Reply:
Thanks @ashb . 
Earlier I was thinking the same, but I agree with Bas's point that some people may have done some hacks using these two endpoints. Will keep them here in case.